### ISSUE TITLE
Is showRatio public, and should we fix related problems? 

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#231]
From https://github.com/phetsims/ph-scale/issues/229#issuecomment-887055145...  With the `showRatio` query parameter, the numeric ratio of H3O+ to OH- is shown in the Micro and My Solution screens, like "15 / 169" in this screenshot:

<img width="552" alt="screenshot_1093" src="https://user-images.githubusercontent.com/3046552/127065415-bdf3014e-5183-4a86-ad10-dd84d9728d1d.png">

This was originally a developer-only feature, to help me determine how many red and blue molecules I was drawing.   In fact, this query parameter used to use be named "dev", and was changed to "showRatio" back in 2016, see 3d3159a.

There are a few problems with making `showRatio` public-facing:

(1) I can't find any GitHub discussion or papertrail about why it was made public.  All of the evidence points to it NOT being public. Looking at the commits for PHScaleQueryParameters.js, I almost certainly made it public by mistake in 45b250c .  And problems (2) and (3) below are things that we are unlikely to have overlooked if we intentionally made this public.  So the first question is, should `showRatio` really be public?  If not, then the other questions are moot.

(2) The ratio is expressed as the actual numbers of molecules, red/blue, e.g. 2/126.  This is valuable for development, but not so good as a public-facing ratio.  A reduced ratio would almost certainly be preferrable, e.g. 2/126 = 1/63.  But if we reduce it, then it becomes sort of useless for development.

(3) The ratio really needs a transparent background. At the pH extremes, the molecules result in (effectively) a red or blue background, and the black numbers become difficult to see. See screenshots below. The numbers should be on a transparent white background.

pH = -1:

<img width="507" alt="screenshot_1091" src="https://user-images.githubusercontent.com/3046552/127064569-c9d09c7f-cb38-48a3-96b3-b92245b6d1ea.png">

pH = 15:

<img width="549" alt="screenshot_1092" src="https://user-images.githubusercontent.com/3046552/127064578-0137dc89-06b5-40ac-acd8-b964d8e06d4b.png">

Unfortunately, `showRatio` was just added to the PhET-iO Guide in #229, and patched into 1.5. So this needs to be addressed before publishing 1.5.   

So...  My recommendation is that `showRatio` should be private (for internal use only), and #229 should be reverted.

**Comment by arouinfar** [comment]
This is a sim that I inherited, so I was not around for the original design discussions. I discovered `showRatio` when looking at public query parameters for the Teacher Tips, and its comments included "For external use." If it was created as an internal testing tool, let's make it private and revert the commits in #229.

**Comment by pixelzoom** [comment]
OK, I'm going to make `showRatio` private, and revert #229.  Like I said above, it became public almost certainly due to an error that I committed in https://github.com/phetsims/ph-scale/commit/45b250c5a859c18df5d470c62b273d6fa45a9c50.

**Comment by pixelzoom** [comment]
`showRatio` is now private in master and 1.5 branches for ph-scale. It's irrelevant in the 1.5 branch for ph-scale-basics.  

I also removed it from [2021 PhET Sim: Query Parameter Review](https://docs.google.com/spreadsheets/d/1TddeTtD8zAayJzs02g0fL7dR1iWYczs1CMFdYd3SBNc/edit).

Closing because there's really nothing for QA to verify here. It's an internal documentation change, and the behavior of the query parameter isn't affected.
